app/grandchallenge/retina_core/management/commands/copyspacingintoimagefields.py

```diff
@@ -0,0 +1,53 @@
+from django.core.management.base import BaseCommand
+from django.core.paginator import Paginator
+
+from grandchallenge.cases.models import Image
+
+
+class Command(BaseCommand):
+    help = "Copy spacing or size from itk header to voxel_(width|height|depth)_mm Image fields"
+
+    def handle(self, *args, **options):
+        images = Image.objects.filter(
+            archive__title__in=[
+                "AREDS - GA selection",
+                "RS1",
+                "kappadata",
+                "Rotterdam_Study_1",
+                "Rotterdam Study 1",
+                "Australia",
+                "RS3",
+                "RS2",
```

Comment:
These will go soon, right? If not, move them into settings?

Reply:
The filtering of archives on their name will be replaced with correct object level permissions in #1296. I probably won't change it in this command however since this is meant to be run once only.

Reply:
:+1: